plugins/modules/rabbitmq_user.py

```diff
@@ -106,6 +106,33 @@
     required: false
     default: on_create
     choices: ['on_create', 'always']
+  protocol:
+    description:
+      - Specify which TCP/IP protocol will be used
+    type: str
+    default: http
+    choices: ['http', 'https']
+  host:
+    description:
+      - Hostname from API
+    type: str
+    required: false
```

Comment:
```suggestion
    required: false
    version_added: '1.2.0'
```
also just a fyi required is false by default, so no need to specify it explicitly.